contracts/abstract/JBOperatable.sol

```diff
@@ -3,6 +3,10 @@ pragma solidity 0.8.6;
 
 import './../interfaces/IJBOperatable.sol';
 
+// --------------------------- custom errors -------------------------- //
+//*********************************************************************//
+error NO_PERMISSION();
```

Comment:
UNAUTHORIZED

Reply:
Would add OPERATABLE in the name, since it's a modifier widely inherited